lib/globber.js

```diff
@@ -1,29 +1,35 @@
 'use strict';
 const glob = require('glob');
 const Rx = require('rx');
+const fs = require('fs');
 
 function globber(paths) {
   const files = new Rx.Subject();
   files.fileEvents = [];
 
-  const Glob = glob.Glob;
   let doneCount = 0;
 
-  paths.forEach(function (path) {
-    const fileEvents = new Glob(path);
+  function globFiles(path) {
+    const fileEvents = new glob.Glob(path);
 
     fileEvents.on('match', function (file) {
-      files.onNext(file);
+      if (fs.statSync(file).isFile()) {
+        files.onNext(file);
+      } else {
+        globFiles(file + '/*');
+      }
```

Comment:
added in f158a25, this `else` block allows including everything inside a matching directory. I don't like it very much but it's the best I've got so far without much complexity.

This means, calling test262-harness from the CLI:

- `test/built-ins/Reflect`, `test/built-ins/Reflect/*`, and `test/built-ins/Reflect/` will catch everything inside that folder.
- `test/built-ins/TypedArray/prototype*` will catch the prototype.js file and everything inside the prototype folder including its subfolders.
- `test/built-ins/Array/**/*` will catch everything including files in `/prototype/` subfolders, recursively.